### PR TITLE
add CloudTrail to description.

### PR DESCRIPTION
**Pull request by sot528**
*Description of changes:*
Add CloudTrail to the description.

### CODE REVIEW

File: doc_source/stacksets-sampletemplates.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # AWS CloudFormation StackSets sample templates<a name="stacksets-sampletemplates"></a>
 
-This section includes links to some sample AWS CloudFormation templates that can help you use AWS CloudFormation StackSets in your enterprise\. Templates listed in this section enable [AWS Config](http://docs.aws.amazon.com/config/latest/developerguide/WhatIsConfig.html) and rules within it\.
+This section includes links to some sample AWS CloudFormation templates that can help you use AWS CloudFormation StackSets in your enterprise\. Templates listed in this section enable [CloudTrail](https://aws.amazon.com/cloudtrail) or [AWS Config](http://docs.aws.amazon.com/config/latest/developerguide/WhatIsConfig.html) and rules within it\.
```
Comment (rachfop) [inline doc_source/stacksets-sampletemplates.md:3]: Hey @sot528 , I would love to merge your PR, could you change the name of CloudTrail to AWS CloudTrail. Thanks!

Comment (rachfop) [reply doc_source/stacksets-sampletemplates.md:3]: And just for consistency, use this url: https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-user-guide.html

For the link to the AWS CloudTrail docs! Thanks.

Comment (sot528) [reply doc_source/stacksets-sampletemplates.md]: OK, Done. 👍

**Comment by rachfop** [comment]
Thank you for your contribution!

**Pull request merged**